aggregator/src/tests/blob.rs

```diff
@@ -298,8 +298,22 @@ fn blob_circuit_completeness() {
         nonempty_chunk_followed_by_empty_chunk,
         empty_and_nonempty_chunks,
         all_empty_except_last,
-    ] {
-        assert_eq!(check_data(BatchData::from(&blob)), Ok(()), "{:?}", blob);
+    ]
+    .into_iter()
+    .enumerate()
+    {
+        let batch_data = BatchData::from(&blob);
+
+        // First blob is purposely constructed to take full blob space
+        if idx == 0 {
+            assert_eq!(
+                batch_data.get_encoded_batch_data_bytes().len(),
+                N_BLOB_BYTES,
+                "should be full blob"
+            );
```

Comment:
Ideally we should also log what's the value if the assertion wasn't met (to avoid re-running the test with additional logging):

```suggestion
            let encoded_len = batch_data.get_encoded_batch_data_bytes().len();
            assert_eq!(
                encoded_len,
                N_BLOB_BYTES,
                "should be full blob: expected={N_BLOB_BYTES}, got={encoded_len}",
            );
```